dropshot/src/server.rs

```diff
@@ -448,7 +448,7 @@ impl TryFrom<&ConfigTls> for rustls::ServerConfig {
             .with_safe_default_kx_groups()
             .with_safe_default_protocol_versions()
             .unwrap()
-            .with_client_cert_verifier(rustls::server::NoClientAuth::new())
+            .with_client_cert_verifier(rustls::server::NoClientAuth::boxed())
```

Comment:
This looks like the only non-test change in this PR aside from the version bump, and it LGTM